src/main/java/org/fundacionjala/coding/abel/movies/Rental.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.coding.abel.movies;
+
+/**
+ * Created by Unknown on 5/26/2017.
+ */
+public class Rental {
+
+    private Movie movie;
+    private int daysRented;
+    private double amount;
+    private int frequentRenterPoints;
+
+    /**
+     * This is the Constructor for Rental.
+     * @param movie Accepts an instance of Movie.
+     * @param daysRented Int days rented.
+     */
+    public Rental(Movie movie, int daysRented) {
+        this.movie = movie;
+        this.daysRented = daysRented;
+        amount = getMovie().calculateAmount(daysRented);
+        frequentRenterPoints = getMovie().calculateFrequentRenterPoints();
```

Comment:
the method to calculate amount and frequentRenterPoints should return those values.